cli/print.go

```diff
@@ -63,6 +71,10 @@ func fatalIf(err *probe.Error, msg string, data ...any) {
 	if err == nil {
 		return
 	}
+	// Clean up UI if one is registered
+	if u := globalUI.Load(); u != nil {
+		u.Wait()
+	}
```

Comment:
It's not ideal, but the problem is fatalIf does exit(). Another solution would be to refactor fatalIf itself, but it's used everywhere...